.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "18.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.11.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
+    name: Webhint
+    runs-on: ubuntu-22.04
+    steps:
```

Comment:
- [ ] Your linter check is not responding because you're missing two files in the root of your README just as pointed out by your last reviewer.

## Your root:
![image](https://user-images.githubusercontent.com/50990925/235429341-15e7dd89-1d69-4380-84e5-6b678cfd8899.png)

## Missing files:
![image](https://user-images.githubusercontent.com/50990925/235429393-fcf894c8-3b07-4d13-b458-ba71318c5ab5.png)

- Use [this link](https://github.com/microverseinc/linters-config/tree/master/html-css) to get the files